content/docs/intro/cloud-providers/_index.md

```diff
@@ -25,8 +25,10 @@ aliases: ["/docs/reference/clouds/"]
 
 ## Infrastructure Providers
 
+* [Consul]({{< relref "./consul" >}})
 * [Docker]({{< relref "./docker" >}})
 * [Kubernetes]({{< relref "./kubernetes" >}})
+* [Vault]({{< relref "./vault" >}})
```

Comment:
I feel like these are in a very different category than Docker and Kubernetes.

And in particular, this ends up *really* burying Kubernetes - which is one of our most important providers.

I think we should probably move Kubernetes back into the "Cloud" section, or create a new section for the "big 4" providers (AWS, Azure, GCP, Kubernetes) to make clear these are the core of what Pulumi is about.

Reply:
@lukehoban I am following the same as per the HC docs - we can restructure these as much as you want - I just need to know what way people want them :)